config/config/sc-config.yaml

```diff
@@ -1081,6 +1081,8 @@ prometheusBlackboxExporter:
     thanosQuery: true
     thanosReceiver: true
 
+kubeapiServers: []
```

Comment:
Could we instead have something like
```yaml
prometheusBlackboxExporter:
  # Informative comment..
  customKubeapiTargets:
    - name: ..
      domain: ..
```
?

Reply:
I like it. [Changed it](https://github.com/elastisys/compliantkubernetes-apps/blob/anders-elastisys/add-multiple-kubeapiservers-support/config/config/sc-config.yaml#L1081-L1086), better?